saleor/order/fetch.py

```diff
@@ -147,4 +153,61 @@ def fetch_draft_order_lines_info(
     ):
         voucher_info = fetch_voucher_info(voucher, order.voucher_code)
         apply_voucher_to_line(voucher_info, lines_info)
```

Comment:
IMHO, this should be embeded in single function: like `apply_voucher_to_editable_order_line_info` or something. 
Inside the body, we should have:
```
apply_voucher_to_line(voucher_info, lines_info)
denormalized_voucher_info = _get_denormalized_voucher_info(lines_info, voucher)
_apply_denormalized_voucher_to_line_info(
             lines_info, denormalized_voucher_info, order.voucher_code
         )
```
Right now you need to call a three function in proper order, to get expected results.